### ISSUE TITLE
Create and Deploy HTML5 and SAP Fiori Launchpad Site Modules

### ISSUE THREAD

**Issue by specialword** [SAPDocuments/Tutorials#4844]
Tutorial URL: https://developers.sap.com/tutorials/abap-environment-deploy-cf-production.html

Please specify the step you are referring to step4.
when you click to select the resource, 
step 5 login screen does not happen.

George

**Comment by mervey45** [comment]
Hi @specialword ,

thank you for your feedback. There seems to be an issue. We are working on this. As soon as we have an answer, we'll update this tutorial.

Thank you for your understanding.

Best regards, 
Merve

**Comment by specialword** [comment]
update:
sometimes, if you drill down to click on MY specific resource, then WAIT, it works as guiding me to the login screen. wait is important because it takes a while and I did not realize that I need to wait... 
then I have problem understand step5, what the url it is to show my application in sap cloud platform. it would be nice if we could tell what the url is because I am new to this, I did try something like 
https://webidecp-p1942041822trial.dispatcher.hanatrial.ondemand.com/

I am going to try step 4 again to find a pattern.

thanks,

George

**Comment by mervey45** [comment]
Hi @specialword ,

which URL do you mean in specific?

Thank you.

Best regards, 
Merve

**Comment by specialword** [comment]
sorry Merve, my mistake.
I mean step 8 , substep 5, 
Logon to your SAP Cloud Platform ABAP environment trial or SAP Cloud Platform ABAP environment.

this url, I am not sure. I assume it is 
https://webidecp-p1942041822trial.dispatcher.hanatrial.ondemand.com/

because of my problem at step 4, I don't see  the tile on step8 substep 6. 

George

**Comment by mervey45** [comment]
Hi @specialword ,

the browser will open automatically when you do step 8.4. Make sure you have for e.g. chrome as default browser. If you are using Internet Explorer it might not work. Use Chrome, Firefox or MS Edge.

Best regards, 
Merve

**Comment by specialword** [comment]
yes Merve, I do use chrome. let me try again.
meanwhile, I see 
now step1 substep3, multi-target template is no longer available. 
![image](https://user-images.githubusercontent.com/52459595/78033384-f0dc9f80-732b-11ea-817e-e9b429e0f757.png)
I think it was temporarily taken away to fix the issue?
appreciate it Merve.
I am going to restart my computer now see if reboot will fix this issue.
George

**Comment by mervey45** [comment]
Hi @specialword 

MTA is there, please try it again.
![image](https://user-images.githubusercontent.com/27953324/78034857-94e43c00-7368-11ea-94cf-bbbf79449d26.png)

Best regards,
Merve

**Comment by mervey45** [comment]
Hi @specialword ,

you selected favorites as a category, that's why you don't see it.

Best regards, 
Merve

**Comment by specialword** [comment]
you are right, Merve. thanks for your help. now I see list report application tile. I am proceeding to try step 4 again.

George

**Comment by specialword** [comment]
sorry I mean multi-target application, not list report application.

**Comment by specialword** [comment]
apologize, Merve. let me be more specific, the problem I have is for step 5 substep4:
![image](https://user-images.githubusercontent.com/52459595/78039629-f9d16f00-7333-11ea-9421-f02fcde6472c.png)

and this issue is now resolved.
if I am NOT logged on, substep 4 will invoke a new tab on chrome requesting login.
the after I logged in, close this tab, go back to sap web ide
at step6, I had problem yesterday, and today so far it seems to be fixed or no longer a problem.
![image](https://user-images.githubusercontent.com/52459595/78039886-4a48cc80-7334-11ea-8f4f-f719fc51091d.png)

when I click my service, do not expand to below DB table list... the "next" button was greyed out yesterday but today it is available. when I click my service, I notice I need to WAIT  for a few seconds . maybe yesterday I didn't wait and next button was grey. .. maybe this is my problem..
